jetty-distribution/pom.xml

```diff
@@ -350,7 +350,7 @@
               <arguments>
                 <argument>jetty.home=${assembly-directory}</argument>
                 <argument>jetty.base=${assembly-directory}/demo-base</argument>
-                <argument>--add-to-startd=server,continuation,deploy,websocket,ext,resources,client,annotations,jndi,servlets,jsp,jstl,http,https</argument>
+                <argument>--add-to-startd=server,continuation,deploy,websocket,ext,resources,client,annotations,jndi,servlets,jsp,jstl,http,https,threadpool</argument>
```

Comment:
@WalkerWatch there is no need to explicitly add the `threadpool` module here because it is a dependency of the `server` module.

Reply:
@sbordet Correct, though I did this explicitly so the `threadpool.ini` would appear in the `demo-base/start.d` directory. While most users will not make changes to this, I wanted it to be available for demonstration purposes.

Reply:
+1 for demo purposes!

Reply:
`start.d/threadpool.ini` should still show up, as its a dependency of `server` with properties that can be configured.

Reply:
@joakime we changed some time ago to only generating ini files for explicitly enabled modules.  When enabling a module with transitive dependencies that can have module files, an info is generated to tell you that they exist.

Reply:
Correct. When enabling a module that has dependencies, the dependent modules are created but their ini files are not. In this instance you'll see:

    $ java -jar ../start.jar --add-to-start=server
    INFO  : server          initialized in ${jetty.base}/start.d/server.ini
    INFO  : threadpool      transitively enabled, ini template available with --add-to start=threadpool
    INFO  : Base directory was modified

Reply:
Seems like a big deal of a change.  (I totally missed it, and i've been paying attention to VERSION.txt).
Know when that happened?

Reply:
The last time the particular line was changed was October 2016.  I think the behaviour was changed a bit before that.   It was a good change as there is no need to create ini files unless they are needed to be changed.

Reply:
@joakime It was done as part of https://github.com/eclipse/jetty.project/issues/984 I believe.

Reply:
Showed up in versions 9.4.0 and onwards.  Good to know.